pyulog/params.py

```diff
@@ -29,6 +29,9 @@ def main():
     parser.add_argument('-t', '--timestamps', dest='timestamps', action='store_true',
                         help='Extract changed parameters with timestamps', default=False)
 
+    parser.add_argument('-q', '--qgc_compatibility', dest='qgc_compatibility', action='store_true',
```

Comment:
The CLI interface gets a bit convoluted. Can you change it to `-f|--format <format>`, where `<format>` is one of  `csv|octave|qgc`? And then document with which options `--timestamps` and `--initial` can be used.

Reply:
Uhh one thing I noticed is the `timestamp` option doesn't seem to do anything. Could you elaborate on the intended use case for `timestamp`, `initial`, and `octave`? I don't really understand what information they are supposed to be conveying.

Reply:
Is that not clear from the param descriptions and the code?
Please test the different options and check the outputs.
`csv|octave|qgc` are output formats and `timestamp|initial` are output options that apply to certain formats.
If it's still not clear then, let me know.